kubectl-plugin/src/operations.rs

```diff
@@ -16,21 +15,24 @@ pub(crate) enum Operations {
 /// To be implemented by resources which support the 'list' operation.
 #[async_trait(?Send)]
 pub trait List {
-    async fn list() -> Result<()>;
+    type Format;
+    async fn list(output: &Self::Format);
```

Comment:
Why is the format a type specifier in these traits, we seem to always just set it to resources::OutputFormat? So could the parameter simply be: `output: &OutputFormat`?

Reply:
Yes, We could have just used the same. Done, Thanks.